src/lib/theme/typography.ts

```diff
@@ -48,4 +67,18 @@ export const TYPOGRAPHY = {
     fontSize: '14px',
     lineHeight: '18px',
   },
+  subtitle1: {
```

Comment:
can it be more descriptive? 
e.g. postCardSubtitle or something.. subtitle1 | subtitle2 is not good, at some point you'd have subtitle22 and how to distinguish them ..

Reply:
As of today, Material UI library V4, which is currently used on the project, provides us with the limited number of Typography default  variants names.